example/app/config.yml

```diff
@@ -61,7 +61,9 @@ default:
   # test configuration ends
 
   cmaDir: '/opt/'
-  cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:4
+  sandbox_cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:4
+  sit_cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:1
+  cma_layer: '{{sandbox_cma_layer}}'
```

Comment:
Because we can't share lambda layers across accounts we have to have separate ones for each account.